src/ui/pages/HardwareWalletConnection/HardwareSignTransaction/HardwareSignTransaction.tsx

```diff
@@ -65,10 +65,12 @@ export const HardwareSignTransaction = React.forwardRef(
       derivationPath,
       isSending,
       children,
+      title,
       ...buttonProps
     }: React.ButtonHTMLAttributes<HTMLButtonElement> & {
       derivationPath: string;
       isSending: boolean;
+      title?: React.ReactNode;
```

Comment:
"title" can be confused with the native title attribute, I'd go for "text" or "label" or "buttonText"